runner/utils/driver/istio/istio.go

```diff
@@ -82,3 +124,7 @@ func checkIfVolumeIsSecret(name string, volumes []v1.Volume) bool {
 func (i *Istio) Type() model.DriverType {
 	return model.TypeIstio
 }
+
+func (i *Istio) getStatusOfDeployement(projectID, deployementID string) bool {
+	return i.waitservice[projectID][deployementID].AvailableReplicas >= 1 && i.waitservice[projectID][deployementID].ReadyReplicas >= 1
```

Comment:
Check for the existense of projectID & deploymentID in the map first, otherwise there is a chance of panic error